includes/mailjet.mail.inc

```diff
@@ -476,28 +481,27 @@ class MailjetSmtpMailSystem implements MailSystemInterface {
 
     // Set other connection settings.
     //$mailer->Host   = variable_get('mailjet_host', 'pp-leela1.mailjet.com');
-    $mailer->Host   = variable_get('mailjet_host', 'in-v3.mailjet.com');
-    $mailer->Port   = variable_get('mailjet_port', '587');
+    $mailer->Host = variable_get('mailjet_host', 'in-v3.mailjet.com');
+    $mailer->Port = variable_get('mailjet_port', '587');
     $mailer->Mailer = 'smtp';
 
     if (variable_get('mailjet_debug')) {
-        watchdog('mailjet', 'Sending mail to: @to', array('@to' => $to));
+      watchdog('mailjet', 'Sending mail to: @to', array('@to' => $to));
     }
 
     // Try to send e-mail. If it fails, set watchdog entry.
     if (!$mailer->Send()) {
       if (variable_get('mailjet_debug')) {
-          watchdog('mailjet',
-                   'Error sending e-mail from @from to @to : @error_message',
-                   array(
-                     '@from' => $from,
-                     '@to' => $to,
-                     '@error_message' => $mailer->ErrorInfo),
-                   WATCHDOG_ERROR);
+        watchdog('mailjet', 'Error sending e-mail from @from to @to : @error_message', array(
```

Comment:
According to PSR-2 - Coding style guide, the first item in the list MUST be on the next line, and there MUST be only one argument per line.